lib/actions/ssh.coffee

```diff
@@ -92,7 +92,7 @@ module.exports =
 			Promise.props
 				username: resin.auth.whoami()
 				uuid: device.uuid		# get full uuid
-				containerId: resin.models.device.getApplicationInfo(device.uuid).get('containerId')
+				containerId: resin.models.device.getApplicationInfo(device.uuid).then(JSON.parse).get('containerId')
```

Comment:
Yes, we used to use `request` inside `resin-request`, and pass `json: true` which forces parsing all response bodies as JSON. We now use the content-type, which is correct behaviour. Sounds like this isn't a new change in the API, it's just been exposed by changes in resin-request.

This needs fixing before the cli can go out, since it'll be broken for any supervisor interactions. The SDK is already out with this so these methods are broken there, but it's only out as a prerelease, so that's not a disaster. Who should fix this? Happy to take a look myself, but I haven't touched the API yet, so I'll need some pointers from somebody.